c++/h5/group.cpp

```diff
@@ -95,6 +95,17 @@ namespace h5 {
     return group(obj, parent_file);
   }
 
+  void group::create_softlink(std::string const &target_path, std::string const& link_name) const {
```

Comment:
Given the naming conventions within h5 it would be more consistent to use `target_key` instead of `target_path` and `key` instead of `link_name`.